vendor/map-parser/ultrasimple-map-parser.ts

```diff
@@ -65,17 +60,6 @@ export class UltraSimpleMapParser {
         return archiveFiles;
     }
 
-    protected async extractSdz(sdzPath: string, outPath: string) {
-        if (!existsSync(sdzPath)) {
-            throw new Error(`File not found: ${sdzPath}`);
-        }
-        await fs.mkdir(outPath, { recursive: true });
-        const zip = new StreamZipAsync({ file: sdzPath });
-        await zip.extract("maps/", outPath);
-        await (zip as any).close();
-        return this.extractArchiveFiles(outPath);
-    }
-
     protected async extractArchiveFiles(outPath: string) {
         const files = glob.sync(`${outPath}/**/*`);
```

Comment:
This doesn't work on windows so no maps load on windows, needs `{ windowsPathsNoEscape: true }`